acceptance/helpers/regex/regex.go

```diff
@@ -0,0 +1,20 @@
+package regex
+
+import (
+	"fmt"
+)
+
+const (
+	// DateRegex will check for a date in the '2022-05-19 13:49:20 +0000' UTC format
+	DateRegex = "[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} [+][0-9]{4} [A-Z]{3,4}"
+)
+
+// TableRow returns a regular expression that will match a line of a CLI table
+// The arguments passed should contain the expected text of the cell (or regex, i.e. you can pass '.*' to match anything)
+func TableRow(args ...string) string {
+	reg := `\|`
+	for _, arg := range args {
+		reg = fmt.Sprintf(`%s[\s]+%s[\s]+\|`, reg, arg)
```

Comment:
If an `arg` is the empty string, i.e. `""`, the resulting regexp will contain `[\s]+[\s]+\|`. I.e. it will expect two spaces. Not sure if that is guaranteed by the table output itself. Might be sensible to check for that case and then use `reg = fmt.Sprintf(`%s[\s]+\|`, reg)` instead. Although that could be written as `reg = reg + `[\s]+\|``.

Reply:
I've tried with a small test and it seems that spaces are always added:

```go
func TestTable(t *testing.T) {
	msg := termui.NewUI().Normal()
	msg.WithTable("", "", "", "", "").WithTableRow("", "").KeepLine().Msg("")
}
```
```
=== RUN   TestTable
|  |  |  |  |  |
|--|--|--|--|--|
|  |  |
--- PASS: TestTable (0.00s)
```

but I've seen that an empty row will not be printed at all

```go
func TestTable(t *testing.T) {
	msg := termui.NewUI().Normal()
	msg.WithTable("", "", "", "", "").WithTableRow().KeepLine().Msg("")
}
```
```
=== RUN   TestTable
|  |  |  |  |  |
|--|--|--|--|--|
--- PASS: TestTable (0.00s)
PASS
```

So at the moment I rewrote it like this (with a guard clause and the `strings.Builder`):

```go
func TableRow(args ...string) string {
	if len(args) == 0 {
		return ""
	}

	var b strings.Builder
	for _, arg := range args {
		fmt.Fprintf(&b, `[|][\s]+%s[\s]+`, arg)
	}
	b.WriteString(`[|]`)

	return b.String()
}
```

What do you think?

Reply:
Ah, now I get what you're saying. Well, it should be true, but it seems to work anyway :thinking: 

https://regex101.com/r/pVSouv/1

Reply:
If you group them you can see that they are going to "share" the spaces.

https://regex101.com/r/ik9bRP/1

Reply:
Testing with ginkgo shows that it works for the go regex engine as well. No bug. Ok.

Reply:
```go
Describe("", func() {
	It("Regex", func() {
		Expect("| yee |  |  |").To(MatchRegexp(regex.TableRow("yee", "", "")))
		Expect("| foo |").To(MatchRegexp(regex.TableRow("foo")))
		Expect("|  |").To(MatchRegexp(regex.TableRow("")))
	})
})
```